{"name": "Guest filters shore excursions results using price range", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: chrome=80.0.3987.163)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\guest_explores_ports_of_departure_steps.py\", line 8, in navigated_shore_excursions\n    context.app.top_navigation.click_shore_excursions()\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\top_navigation.py\", line 17, in click_shore_excursions\n    self.wait_for_element_click(*self.SHORE_EXCURSIONS)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\base_page.py\", line 39, in wait_for_element_click\n    e.click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given a Guest", "status": "passed", "start": 1588302327586, "stop": 1588302337479}, {"name": "And I am on Homepage", "status": "passed", "start": 1588302337479, "stop": 1588302338310}, {"name": "And I navigated to \"Shore Excursion\" page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: chrome=80.0.3987.163)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\guest_explores_ports_of_departure_steps.py\", line 8, in navigated_shore_excursions\n    context.app.top_navigation.click_shore_excursions()\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\top_navigation.py\", line 17, in click_shore_excursions\n    self.wait_for_element_click(*self.SHORE_EXCURSIONS)\n  File \"C:\\Everything\\My_docs\\Self_development\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\base_page.py\", line 39, in wait_for_element_click\n    e.click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1588302338311, "stop": 1588302343361}, {"name": "And I click Find Excursions", "status": "skipped", "start": 1588302345685, "stop": 1588302345685}, {"name": "And Shore Excursions page is present", "status": "skipped", "start": 1588302345685, "stop": 1588302345685}, {"name": "When Price range is filtered to \"$0-$30\"", "status": "skipped", "start": 1588302345685, "stop": 1588302345685}, {"name": "Then Only shore excursions within range are displayed", "status": "skipped", "start": 1588302345685, "stop": 1588302345685}], "start": 1588302318345, "stop": 1588302345685, "uuid": "a03b8ef9-4d0a-41e3-a8b2-bc2b66d4f173", "historyId": "0ee61ef5695883abbe93102197813db0", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Guest filters shore excursions results using price range"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}